### ISSUE TITLE
Upgrade from 12.109.2 to 12.110.0 fails with error 

### ISSUE THREAD

**Issue by memo-567** [YunoHost-Apps/misskey_ynh#69]
Misskey upgrade fails on productive server (Debian 10 Buster), but not on test server (VirtualBox test environment).

```
[Error] Upgrade failed.
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
This action broke dpkg/APT (the system package managers)... You can try to solve this issue by connecting through SSH and running `sudo apt install --fix-broken` and/or `sudo dpkg --configure -a`.
```

No changes after runnig 'sudo apt install --fix-broken': 0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
and 'sudo dpkg --configure -a' 

Second upgrade also fails.

Logfile: https://paste.yunohost.org/raw/olidifival

**Comment by memo-567** [comment]
I uninstalled the two apps (Plume and Invidious) for testing because they also access the Postgres server. No changes.

**Comment by memo-567** [comment]
Also, no changes after @jarod5001 PR.

**Comment by memo-567** [comment]
Just a thought: One difference between my productive server and the Virtual-Box YunoHost installation is that all Debian packages are up-to-date on the productive server.

The Virtual-Box YunoHost installation was tested without Debian updates.

**Comment by memo-567** [comment]
I don't know if it's relevant, but I updated Yarn just before:

Upgrade: yarn:amd64 (1.22.17-1, 1.22.18-1)

**Comment by memo-567** [comment]
Johann (misskey-dev) sent a hint, this might be an issue  in TypeORM:

https://github.com/typeorm/typeorm/issues/8885

**Comment by memo-567** [comment]
This solution might help:

https://github.com/misskey-dev/misskey/issues/8487#issuecomment-1099579927

But the upgrade script would have to be revised for that?

**Comment by ericgaspar** [comment]
should be fixed with #73
